capstone/src/test/java/com/google/univiz/SuggestionDataTest.java

```diff
@@ -0,0 +1,28 @@
+package com.google.univiz;
+
+import static com.google.common.truth.Truth.assertThat;
+
+import com.google.common.io.Resources;
+import com.google.gson.Gson;
+import com.google.gson.GsonBuilder;
+import com.ryanharter.auto.value.gson.GenerateTypeAdapter;
+import java.io.InputStreamReader;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+
+@RunWith(JUnit4.class)
+public final class SuggestionDataTest {
+  /** Tests whether the JSON properly deserializes into a SuggestionData object */
+  @Test
+  public void testJsonDeserializes() throws Exception {
```

Comment:
Are we ever going to deserialize this json? Might make more sense to verify that it serializes as expected, since we'll be writing that JSON to the client :)

Reply:
I think you're confusing SuggestionData with SearchData (as i did before). Or maybe i'm the confused one right now 😂. So this one is the one that we deserialize the scorecard results into. SearchData is the one our servlet returns.

Reply:
Resolved in standup :)